deploy/README.md

```diff
@@ -333,6 +327,16 @@ It is possible to migrate an existing Wikibase installation to WBS Deploy. The g
  - Regenerate the WDQS database
  - Regenerate the Elasticsearch database
 
+### Can I host WBS Deploy locally?
+
+Due to the OAuth configuration for MediaWiki and QuickStatements, along with the automatic SSL certification generated by Traefik, you must specify values for `WIKIBASE_PUBLIC_URL` and `QUICKSTATEMENTS_PUBLIC_URL` in your `.env` file. These values should be domain names that route to the IP address of the server hosting these services and must be accessible on the Internet.
+
+However, for testing purposes WBS Deploy can be run locally on a server that is not accessible to the Internet, with the following caveats:
+
+* In this configuration, you will still need to set `WIKIBASE_PUBLIC_URL` and `QUICKSTATEMENTS_PUBLIC_URL` to URLs that resolve locally to the IP address of the machine running the services. Configuring locally resolving DNS entries differs depending on your environment (Linux, MacOS, Windows), so setting this up correctly will require knowledge of or additional research about your specific setup. 
```

Comment:
> set `WIKIBASE_PUBLIC_URL` and `QUICKSTATEMENTS_PUBLIC_URL`

Are you referring to the variables for the quickstatements container? Couldn't the user still just configure `WIKIBASE_PUBLIC_HOST` and `QUICKSTATEMENTS_PUBLIC_HOST` in `.env`?

Reply:
Yes, the assumption was that these values were being set in the .env not the docker-compose file, but that could be made again explicit here.